programs/server/Server.cpp

```diff
@@ -874,8 +874,7 @@ int Server::main(const std::vector<std::string> & /*args*/)
         };
 
         /// This object will periodically calculate some metrics.
-        AsynchronousMetrics async_metrics(*global_context,
-            config().getUInt("asynchronous_metrics_update_period_s", 60));
+        AsynchronousMetrics async_metrics(*global_context, config().getUInt("asynchronous_metrics_update_period_s", 60), servers);
```

Comment:
If we will move it below, maybe we can get rid of the separate `start` method in AsynchronousMetrics (see added comments)?

Reply:
servers rely on async_metric during creation.

Reply:
Ok.